app/prism/Prism.scala

```diff
@@ -31,6 +31,18 @@ object Prism {
 
   def getInstances(stackStageApp: SSA)(implicit config: AMIableConfig, ec: ExecutionContext): Attempt[List[Instance]] = {
     val url = instancesUrl(stackStageApp, config.prismUrl)
+    getInstancesFromUrl(url)
+  }
+
+  def imageUsage(imageOpt: Option[AMI])(implicit config: AMIableConfig, ec: ExecutionContext): Attempt[List[Instance]] = {
```

Comment:
Why is the argument an Optional? What about dealing with getting or not an AMI outside of this function?